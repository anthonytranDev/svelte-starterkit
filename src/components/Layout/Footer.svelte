<style>
  .footer {
    padding: 1rem 0.75rem;
  }

  .footer-wrapper {
    display: flex;
    justify-content: space-between;
  }

  .footer-license {
    font-weight: 700;
  }

  #github-logo {
    position: relative;
    top: 3px;
    height: 20px;
  }
</style>

<div class="footer" data-testid="Footer">
  <div class="footer-wrapper">
    <div class="footer-license">MIT License</div>
    <div>
      <a href="https://github.com/anthonytranDev">
        <img
          id="github-logo"
          src="/github-logo.svg"
          class="home-logo"
          alt="logo"
        />
        <span>GitHub: anthonytranDev/svelte-basic-starter</span>
      </a>
    </div>
  </div>
</div>
