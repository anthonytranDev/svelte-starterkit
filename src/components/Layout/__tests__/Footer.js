import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/svelte"

import Footer from "../Footer.svelte"

describe("Footer", () => {
  it("should render", () => {
    const { getByTestId } = render(Footer)

    expect(getByTestId("Footer")).toMatchSnapshot()
  })
})
