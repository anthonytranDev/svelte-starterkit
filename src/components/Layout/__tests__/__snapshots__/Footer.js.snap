// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<div
  class="footer svelte-ds2ccu"
  data-testid="Footer"
>
  <div
    class="footer-wrapper svelte-ds2ccu"
  >
    <div
      class="footer-license svelte-ds2ccu"
    >
      MIT License
    </div>
     
    <div>
      <a
        href="https://github.com/anthonytranDev"
      >
        <img
          alt="logo"
          class="home-logo svelte-ds2ccu"
          id="github-logo"
          src="/github-logo.svg"
        />
         
        <span>
          GitHub: anthonytranDev/svelte-basic-starter
        </span>
      </a>
    </div>
  </div>
</div>
`;
